misc/services/mobile-webui/mobile-webui-frontend/src/serviceWorkerRegistration.js

```diff
@@ -40,7 +42,9 @@ export function register(config) {
     } else {
       // Is not localhost. Just register service worker
       console.log('[ServiceWorker] - Registering on Prod server');
-      registerValidSW(swUrl, config);
+      if (REGISTER_SERVICE_WORKER) {
```

Comment:
why we need this here when we already have an `if (REGISTER_SERVICE_WORKER)` then call this `register` function.